package.json

```diff
@@ -71,11 +71,11 @@
     "@dcl/mini-rpc": "^1.0.7",
     "@dcl/schemas": "^11.10.4",
     "@dcl/sdk": "^7.7.0",
+    "@ethersproject/hash": "^5.7.0",
```

Comment:
This is used here: https://github.com/decentraland/creator-hub/blob/35c3e85ff5ce0c839011a3fa0dd9bc7d28842091/packages/renderer/src/modules/store/ens/slice.ts#L3

But it was not installed because it was a dependency of `decentraland-transactions`, so now it needs to be added as a dependency of the Creator Hub